examples/api-dotenv-request/app/Main.hs

```diff
@@ -0,0 +1,31 @@
+import Configuration.Dotenv (loadFile, defaultConfig)
+import System.Environment (lookupEnv)
+import Network.HTTP.Req
+import qualified Data.ByteString.Char8 as B
+import qualified Data.Text as T
+import Data.Aeson
+
+main :: IO ()
+main = do
+    loadFile defaultConfig
+    
+    --  Loading API URI from dotenv 
+    maybeApiEndpoint <- lookupEnv "API_URI"
```

Comment:
where is this env value? I think you should provide a sample file for this example. wdyt?